mlflow/pipelines/regression/v1/pipeline.py

```diff
@@ -523,6 +578,19 @@ def read_dataframe(artifact_name, output_dir, file_name, step_name):
                 log_artifact_not_found_warning("registered_model_version", register_step.name)
                 return None
 
+        elif artifact_name == "ingested_scoring_data":
```

Comment:
Would `scoring_data` be sufficient here?

Reply:
I can change this.

Reply:
Oh.  I think I had this this way because we have a `scored_data` artifact which is the output of the predict step.

Reply:
`ingested_scoring_data` is fine. It just feels a bit long. Let's keep it as is.